client/src/commons/DatePick/DatePick.tsx

```diff
@@ -43,4 +45,6 @@ interface Props extends StandardTextFieldProps {
   value: ParsableDate;
   onChange: any;
   testId?: string;
+  onBlur?: any;
```

Comment:
Why onBlur is optional? We want every DatePick to have an onBlur don't we?

Reply:
eventually yes, at the moment exposures and flights tab stayed as is and not under any validations or controllers, which allows his date pickers to exist without an on blur function (in the future it will be added and onBlur won't include ?)